src/main/java/org/opentripplanner/framework/lang/StringUtils.java

```diff
@@ -1,12 +1,22 @@
 package org.opentripplanner.framework.lang;
 
+import java.util.regex.Pattern;
 import javax.annotation.Nonnull;
 
 /**
  * OTP String utils extending the Java lang String...
  */
 public class StringUtils {
 
+  /**
+   * Regex to find unprintable characters like newlines and 'ZERO WIDTH SPACE' (U+200B).
+   */
+  private static final String INVISIBLE_CHARS_REGEX = "\\p{C}";
```

Comment:
Consider: `\\p{C}` is not documented in the SDK JavaDoc - would it be suffisient to use `\\p{Cntrl}`?

Reply:
Will add docs.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5703/commits/340ca13b7d2a132d3875c4da475855727b970b01